src/content-layout/__tests__/content-layout.test.tsx

```diff
@@ -102,14 +145,129 @@ describe('ContentLayout component', () => {
+      });
+      expect(wrapper.findByClassName(styles.background)!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findHeader()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findNotifications()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findBreadcrumbs()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+      expect(wrapper.findSecondaryHeader()!.getElement()).not.toHaveClass(highContrastHeaderClassName);
+
+      expect(wrapper.findByClassName(styles['header-wrapper'])!.getElement()).toHaveClass(styles['with-divider']);
+    });
+  });
+
+  describe('maxContentWidth', () => {
+    const halfGeckoMaxCssLength = ((1 << 30) - 1) / 120;
+    test('is set to maximum value if not specified', () => {
+      const { wrapper } = renderContentLayout({
+        header: <>Header text</>,
+      });
+      const maxWidthInGrid = wrapper.getElement().style.getPropertyValue(customCssProps.contentLayoutMaxContentWidth);
+      expect(maxWidthInGrid).toBe(`${halfGeckoMaxCssLength}px`);
+    });
+
+    test('is set to maximum value when maxContentWidth=Nuber.MAX_VALUE', () => {
+      const { wrapper } = renderContentLayout({
+        header: <>Header text</>,
+        maxContentWidth: Number.MAX_VALUE,
+      });
+      const maxWidthInGrid = wrapper.getElement().style.getPropertyValue(customCssProps.contentLayoutMaxContentWidth);
+      expect(maxWidthInGrid).toBe(`${halfGeckoMaxCssLength}px`);
+    });
+
+    test('is set correctly when numeric value is specified', () => {
+      const { wrapper } = renderContentLayout({
+        header: <>Header text</>,
+        maxContentWidth: 1200,
+      });
+      const maxWidthInGrid = wrapper.getElement().style.getPropertyValue(customCssProps.contentLayoutMaxContentWidth);
+      expect(maxWidthInGrid).toBe('1200px');
+    });
+  });
+
+  describe('defaultPadding', () => {
+    test('adds defaultPadding className when specified', () => {
+      const { wrapper } = renderContentLayout({
+        header: <>Header text</>,
+        defaultPadding: true,
+      });
+      expect(wrapper.getElement()).toHaveClass(styles['default-padding']);
+    });
+    test('does not add defaultPadding className when not specified', () => {
+      const { wrapper } = renderContentLayout({
+        header: <>Header text</>,
+        secondaryHeader: <>Secondary text</>,
+      });
+      expect(wrapper.getElement()).not.toHaveClass(styles['default-padding']);
+    });
+  });
+
+  describe('defaultBackgroundStyle', () => {
+    test('adds style property when string is specified', () => {
+      const { wrapper } = renderContentLayout({
+        headerBackgroundStyle: 'blue',
+      });
+      const headerBackgroundStyle = wrapper
+        .findByClassName(styles['header-background'])!
+        .getElement()
+        .getAttribute('style');
+      expect(headerBackgroundStyle).toBe('background: blue;');
```

Comment:
There is `toHaveStyle()` matcher to make this test more clear

Reply:
thanks, applied